pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -22,6 +22,30 @@ package pulsar.proto;
 option java_package = "org.apache.pulsar.common.api.proto";
 option optimize_for = LITE_RUNTIME;
 
+message Schema {
+	enum Type {
+		Json = 1;
+		Protobuf = 2;
+		Thrift = 3;
+		Avro = 4;
+		Reserved1 = 5;
```

Comment:
I don't think we should need these. When deserializing an unknown enum value, protobuf returns `null`. 

Also, we typically we use peer protocol version, which is exchanged in the `Connect`/`Connected` phase. There we can check if the other side supports already a new feature or not.